### DISCUSSION TITLE
Installing Open Shell in Win 10 pre upgrade to Win 11?

### DISCUSSION THREAD

**Discussion by yorktraf** [Open-Shell/Open-Shell-Menu#987] [Q&A] [answered]
I have been running Classic Shell predecessor since we got our Win 10 computer in 2019. I’m considering accepting the offer of free “upgrade” to Win 11 from Microsoft but do not want to lose my classic windows interface including the old (Win 7) look and feel throughout my system.

If I install Open Shell now, before a Win 11 update to our system, will Open shell program and system configuration work after the update? Do I have to uninstall Classic Shell to install Open Shell? I’m worried about that–I don’t know how to navigate the newer Windows interfaces and don’t want to start doing so now, even briefly.

**Comment by tallman69** [comment, marked as answer]
I am running Open Shell v4.4.169 beta on my Windows 11 laptop.  When I got the laptop it was new and had Windows 10 preinstalled on it.  I upgraded to Windows 11 before installing Open Shell so I don't have any experience with your situation.  But I do know that in order to get Windows 11 to work like Windows 7/10 with Open Shell you will have to do some registry tweaks.  And if you like using the quick launch bar, like I do, you will need these tweaks since quick launch is not available in Windows 11 out of the box.  ASKVG has some good guides on some registry tweaks you may want to do.

https://www.askvg.com/tip-restore-old-classic-start-menu-in-windows-11/

Instead of doing the registry tweaks manually I chose to install ExplorerPatcher which does the tweaks automatically for you.   It has a lot of configuration options including changing from the new Windows 11 context menus to the older style menus.  

https://github.com/valinet/ExplorerPatcher

If I were you, I would upgrade your Windows 10 build to the 4.4.169 version of Open Shell and backup your Open Shell settings so you can import them if needed once you get Open Shell up and running in Windows 11.  Then upgrade to Windows 11.  From there I am afraid you will have to deal with the new Windows 11 look until you can get the registry tweaks  done that you will need to do to get back to the Windows 7 feel that you are accustomed to.  I used the search bar a lot to find the Windows 11 settings I needed to help me get back to the Windows 7 feel that I like.  Hope all goes well!

**Reply by yorktraf** [reply]
Thanks very much.

**Reply by tallman69** [reply]
You are welcome!
